test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -733,8 +730,9 @@ class CustomPluginOption
 };
 
 TEST_F(OpenTelemetryPluginOptionEnd2EndTest, Basic) {
-  std::vector<std::unique_ptr<grpc::experimental::OpenTelemetryPluginOption>>
-      plugin_option_list;
+  std::vector<
+      std::unique_ptr<grpc::internal::InternalOpenTelemetryPluginOption>>
+      plugin_option_list(1);
```

Comment:
Passing a size to the ctor is the equivalent of calling `resize()`, not `reserve()` -- it actually adds that number of default-constructed elements to the vector.  So what you're doing here is constructing a vector with 2 elements, the first of which is set to null.  I don't think that's what you intended.

Same for all tests.

Reply:
Of course! Don't know what I was thinking! Thanks!